pkg/apis/servicecatalog/types.go

```diff
@@ -145,10 +145,15 @@ type CommonServiceBrokerSpec struct {
 // ServiceClass allowed property names:
 //   name - the value set to [Cluster]ServiceClass.Name
 //   externalName - the value set to [Cluster]ServiceClass.Spec.ExternalName
+//   externalID - the value set to [Cluster]ServiceClass.Spec.externalID
+//   serviceBrokerName - the value set to [Cluster]ServiceClass.Spec.serviceBrokerName
 //
 // ServicePlan allowed property names:
```

Comment:
The property name "serviceBrokerName" is also allowed for ServicePlan, right?

Reply:
Yeah, I think so.